atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -270,40 +135,11 @@ def group_manifests_and_tag(self, session, worker_digests):
         # And store the manifest list in the push_conf
         push_conf_registry = self.workflow.push_conf.add_docker_registry(session.registry,
                                                                          insecure=session.insecure)
-        for image in self.workflow.tag_conf.images:
+        for image in non_floating_images:
             push_conf_registry.digests[image.tag] = digest
 
         self.log.info("%s: Manifest list digest is %s", session.registry, digest_str)
-        return registry_image.get_repo(explicit_namespace=False), digest
-
-    def tag_manifest_into_registry(self, session, worker_digest):
-        """
-        Tags the manifest identified by worker_digest into session.registry with all the
-        configured tags found in workflow.tag_conf.
-        """
-        self.log.info("%s: Tagging manifest", session.registry)
-
-        digest = worker_digest['digest']
-        source_repo = worker_digest['repository']
-
-        image_manifest, _, media_type, _ = self.get_manifest(session, source_repo, digest)
-        if media_type == MEDIA_TYPE_DOCKER_V2_SCHEMA2:
-            digests = ManifestDigest(v1=digest)
-        elif media_type == MEDIA_TYPE_OCI_V1:
-            digests = ManifestDigest(oci=digest)
-        else:
-            raise RuntimeError("Unexpected media type found in worker repository: {}"
-                               .format(media_type))
-
-        push_conf_registry = self.workflow.push_conf.add_docker_registry(session.registry,
-                                                                         insecure=session.insecure)
-        for image in self.workflow.tag_conf.images:
-            target_repo = image.to_str(registry=False, tag=False)
-            self.store_manifest_in_repository(session, image_manifest, media_type,
-                                              source_repo, target_repo, tag=image.tag)
-
-            # add a tag for any plugins running later that expect it
-            push_conf_registry.digests[image.tag] = digests
+        return registry_image.get_repo(explicit_namespace=False), digest, json.loads(list_json)
```

Comment:
We definitely want to return the *bytes* of the manifest list, not the JSON representation, i.e. `list_json` not `json.loads(list_json)`. Once we've stored it in the registry once, the representation in bytes has been fixed. Any change from that and the digest will be wrong.

Reply:
Fixed.